fenix/app/src/main/java/org/mozilla/fenix/browser/TabPreview.kt

```diff
@@ -45,6 +46,14 @@ class TabPreview @JvmOverloads constructor(
             )
         }
 
+        if (IncompleteRedesignToolbarFeature(context.settings()).isEnabled) {
```

Comment:
We could change this block to 
```suggestion
       val isNavBarEnabled = IncompleteRedesignToolbarFeature(context.settings()).isEnabled)
       binding.tabButton.visibility = !isNavBarEnabled
       binding.tabButton.visibility = !isNavBarEnabled
```